timesketch/frontend/src/components/Common/UploadForm.vue

```diff
@@ -36,10 +36,34 @@ limitations under the License.
         </div>
       </div>
       <div class="field">
-        <span v-if="error">
-          {{ error }}
-        </span>
+        <div v-if="error">
+          <article class="message is-danger mb-0">
+            <div class="message-body">
+              {{ error }}
+            </div>
+          </article>
+        </div>
+      </div>
+
+      <!--- 
+      
+      Next lines of code represent the dropdwon menu
+      It is dynamically generated (with an extra option: New header) to allow
+        the user to map the missing header with an exsting one
+
+      --->
+      <div v-for="(value, key) in hMissing">
```

Comment:
HMissing - is that a typo? If it is for headerMissing I suggest to write it out.